### PR TITLE
bumping k8s versions in test files

### PR DESCRIPTION
**Pull request by chrislovecnm**
updating k8s versions in test YAML files so that we do not get warnings during testing

### CODE REVIEW

File: cmd/kops/integration_test.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func TestComplex(t *testing.T) {
 func TestMinimalCloudformation(t *testing.T) {
 	//runTestCloudformation(t, "minimal.example.com", "../../tests/integration/minimal", "v1alpha0", false)
 	//runTestCloudformation(t, "minimal.example.com", "../../tests/integration/minimal", "v1alpha1", false)
-	runTestCloudformation(t, "minimal.example.com", "../../tests/integration/minimal", "v1alpha2", false)
+	runTestCloudformation(t, "minimal.example.com", "../../tests/integration/cloudformation", "v1alpha2", false)
```
Comment (justinsb) [inline cmd/kops/integration_test.go]: minimal is the only one where we actually test CF output - in theory we should test them all, but I'm not convinced it is worth it.

**Comment by justinsb** [comment]
Do we have to split off the cloudformation directory?  I don't understand why, and it isn't entirely right in that we actually should have a CF test in every dir, just that I think we get most of the value from a TF test

**Comment by chrislovecnm** [comment]
The other tests are far more flexible.  For code maintainability breaking out the CF stuff makes sense in my head.  Put the complex stuff into its own area, so a new developer is not trying to figure out which test is broken and why.

I can move it back, but would rather iterate on it.  I have another PR to bump the versions, which requires more edits to the CF stuff.

It made sense to me to break it out, but I can put it back if it is a show stopper for you.

**Pull request merged**